src/main/java/com/ukubuka/core/reader/UkubukaReader.java

```diff
@@ -81,4 +85,55 @@ public String readFileAsString(final SupportedSource source,
             throw new ReaderException(ex);
         }
     }
+
+    public String readXMLAsString(final SupportedSource source,
+                                   final String completeFileName)
+            throws ReaderException {
+        try {
+            XMLInputFactory inputFactory = XMLInputFactory.newInstance();
+            InputStream in = new FileInputStream(source == SupportedSource.URL
```

Comment:
There should be a better way to get read file as IS. Extract this logic in a separate method.

Reply:
I have extracted the changes to separate method. I think IS is working well.